.nvmrc

```diff
@@ -0,0 +1 @@
+v8
```

Comment:
Is this file specifically for Travis or for using with `nvm use`? Also, should we also add this version to `package.json`?

Reply:
No it's not but Travis does use nvm to set the node version. I tend to work with different node versions across different projects and it's always nice to guarantee I'm using the correct version or switch to it with a simple terminal command. The engine version in package.json should be the minimum version supported.